archfi

```diff
@@ -789,6 +799,46 @@ selectmirrorsbycountry() {
 		sed "s/^\(Server .*\)/\#\1/;/^## $country/N; {s/^\(## .*\n\)\#Server \(.*\)/\1Server \2/}" < /etc/pacman.d/mirrorlist.backup > /etc/pacman.d/mirrorlist
 }
 
+filtermirrorlist() {
+		command -v reflector > /dev/null || { 
+			echo "pacman -S --noconfirm reflector"
+			pacman -S --noconfirm reflector
+		}
+		options=()
+		options+=("${txtfiltermirrorlist}" "")
+		options+=("${txtedit//%1/reflector.conf}" "")
+		sel=$(whiptail --backtitle "${apptitle}" --title "${txtinstallmenu}" --menu "" --cancel-button "${txtback}" --default-item "Filter mirrorlist" 0 0 0 \
+                	"${options[@]}" \
+                	3>&1 1>&2 2>&3)
+		case ${sel} in
+			"${txtfiltermirrorlist}")
+				clear
+				echo "systemctl start reflector.service"
+				systemctl start reflector.service
+				pressanykey
+			;;
+			"${txtedit//%1/reflector.conf}")
+				${EDITOR} /etc/xdg/reflector/reflector.conf
+				filtermirrorlist
+			;;
+		esac		
+}
+
+paralleldownloads() {
+		numpkg=$(whiptail --backtitle "${apptitle}" --title "${txtparallelprompt}" --inputbox "" 0 0 "5" 3>&1 1>&2 2>&3)
+                if [ "$?" = "0" ]; then
+			if [[ "$numpkg" =~ ^[1-9][0-9]*$ ]] ; then
+				sed -i "s/.*ParallelDownloads.*/ParallelDownloads = ${numpkg}/" /etc/pacman.conf
```

Comment:
Works great for me, but I would have also expected that `ParallelDownloads` is set in the actual system. This probably wasn't added since the command runs before pacstrap, but could something like this work?

```suggestion
				sed -i "s/.*ParallelDownloads.*/ParallelDownloads = ${numpkg}/" /etc/pacman.conf
				mkdir /mnt/etc && cp /etc/pacman.conf /mnt/etc
```

Or maybe a second option in the configuration menu after pacstrap?

Reply:
If we just copy the files over, people might have issues with outdated ISO's and stuff, so instead we could just update `pacman.conf` right after after pacstrap finishes. I think this is better.

Reply:
Oh yeah that sounds better